servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -4066,7 +4066,7 @@ void RendererSceneRenderRD::render_scene(RID p_render_buffers, const CameraData
 
 	render_state.voxel_gi_count = 0;
 
-	if (rb != nullptr) {
+	if (rb != nullptr && is_dynamic_gi_supported()) {
 		if (rb->sdfgi) {
```

Comment:
if `is_dynamic_gi_supported()` returns false, `rb->sdfgi` is never initialised and should be `NULL` so this probably isn't needed as we're checking for sdfgi being valid. That said, this is more clear in explaining we're skipping this section if we don't support GI.

Reply:
because a few lines below it also calls `gi.setup_voxel_gi_instances`. GI is not jus SDFGI.